### ISSUE TITLE
Support displacement

### ISSUE THREAD

**Issue by zumpchke** [deadsy/sdfx#33]
Is it possible to support displacement like the torus displacement example?

https://www.iquilezles.org/www/articles/distfunctions/distfunctions.htm

**Comment by deadsy** [comment]
Yes it's possible - because that sort of thing doesn't give an exact distance field you'd probably have to use the slower version of the renderers in order to get a manifold stl.
